source/blog/2018-02-12-ember-3-0-released.md

```diff
@@ -0,0 +1,250 @@
+- [Ember debug function options](/deprecations/v2.x/#toc_ember-debug-function-options)
+
+#### Legacy data binding API
+
+For a long time, Ember has provided two systems for applications to perform data-binding in JavaScript. The first uses computed properties and it is still idiomatic today. For example, you’ll often see `Ember.computed.alias` in application code. This system remains without any deprecation.
+
+The second system uses the `Ember.Binding` namespace or any property with the postfix of `Binding` on an object. For example:
+
+```js
+export default Ember.Component.extend({
+  wowBinding: 'MyApp.wowObject',
+  thingContainer: {},
+  amazingBinding: Ember.Binding.oneWay('thingContainer.firstAmazingThing'),
+  wasThisReallyTheAPIBinding: 'thingContainer.yesThisWasTheAPI'
+});
+```
+
+This second system will be removed in Ember 3.0. For more details, see the deprecation guide:
+
+- [`Ember.Binding`](/deprecations/v2.x/#toc_ember-binding)
+
+#### Legacy {{render}} helper
+
+Ember’s `{{render}}` helper presaged the idea of component-driven rendering architecture as we now know it. Despite this, it was also an API coupled to Ember’s `Controller` system, was subject to edge cases where the context of a template was difficult to discern, and additionally was entangled with Ember’s routing system. Through the 2.x cycle we’ve worked to ensure each use-case for `{{render}}` has a viable solution in Ember’s component-driven system.
+
+Fully deprecated since Ember 2.11, in 3.0 the `{{render}}` helper will be removed. Please see the deprecation guide for more details:
+
+- [`{{#render}}` helper with a block](/deprecations/v2.x/#toc_render-helper-with-block)
+- [Model param in `{{render}}` helper](/deprecations/v2.x/#toc_model-param-in-code-render-code-helper)
+- [`{{render}}` helper](/deprecations/v2.x/#toc_code-render-code-helper)
+- [Rendering into a `{{render}}` helper that resolves to an `{{outlet}}`](/deprecations/v2.x/#toc_rendering-into-a-render-helper-that-resolves-to-an-outlet)
+
+#### Legacy Ember.K utility
+
+Ember has historically provided a short-to-type noop function for use as a stub on parent classes or in other situations where a noop might be useful. With the advent of ECMAScript 2015 [method definitions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions) and [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), this kind of utility has become more confusing than convenient.
+
+In Ember 3.0 the `Ember.K` function will be removed. For more details, please see the deprecation guide:
+
+- [`Ember.K`](/deprecations/v2.x/#toc_code-ember-k-code)
+
+#### Consuming Ember through Bower
+
+Ember.js 3.0 and later releases will no longer be published to bower. If you still use bower today, we strongly suggest you begin the migration of your dependencies to npm.
+
+<hr>
```

Comment:
All of the above from previous comment copied from Road to Ember 3.0 with minor removals for brevity.